sdk/Cargo.toml

```diff
@@ -25,6 +25,7 @@ file_io = ["openssl"]
 serialize_thumbnails = []
 xmp_write = ["xmp_toolkit"]
 no_interleaved_io = ["file_io"]
+external_manifest = ["xmp_write", "file_io"]
```

Comment:
Please also add a description of this new feature in README.

Reply:
I will remove for now since I am not gating anything by this yet.  When reading is enabled it will be used.